docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -390,9 +390,9 @@ sets:
 id: other users names
 question: |
   % if al_form_type in ['starts_case','existing_case','appeal']:
-  What is the name of the ${ ordinal(i) } person who is part of the case with you?
+  Who is the ${ ordinal(i) } person on your side of the case?
   % else:
-  What is the name of the ${ ordinal(i) } person who is adding their name to
+  What is the name of the ${ ordinal(i - 1) } person who is adding their name to
```

Comment:
We should use [`ordinal_number`](https://docassemble.org/docs/functions.html#ordinal_number) here. Apparently, [`ordinal` is just `ordinal_number(i + 1)`](https://docassemble.org/docs/functions.html#ordinal), which is recommended for indices starting with 0. But these questions aren't used for `users[0]`, so `ordinal_number is the better choice.